apps/mobile/src/navigation/linking/index.js

```diff
@@ -128,8 +128,9 @@ export const routingConfig = {
   '/:context(groups)/:groupSlug/members':                                 `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Members`,
   '/:context(groups)/:groupSlug/members/:id':                             `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Member`,
   // TODO: Routing -- Actually legacy redirection, consider creating or adding to a redirect mapper
-  '/:context(groups)/:groupSlug/topics/:topicName':                       `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Chat Room`,
```

Comment:
can you explain this change to me?

Reply:
I don't know what is correct, and this is probably apparent here... but this means that `/:context(groups)/:groupSlug/topics/:topicName' routing now goes to the Stream instead of Chat, which I assume is correct now. If so please wondering if the TODO still is correct or if not if it should be removed?

Reply:
Yep, /chat goes to chat and /topics goes to Stream